### DISCUSSION TITLE
MercatorCoordsArray Center Lon inverted?

### DISCUSSION THREAD

**Discussion by micr0n0va** [cables-gl/cables_docs#947] [Answers & Questions]
Working through the CSV data visualization 3  tutorial and when I reach the map centering technique using InfoArray2 with MercatorCoordsArray it appears that the expected values for Center Lon have inverted, expected value of 19.12 is a miss, but -160.87 gets the map nearly centered. [https://cables.gl/p/a2ogSw](url)

Am I missing something simple?

**Comment by steam0r** [comment]
hm, not sure, your patch is private, though. can't take a look, sorry.

https://cables.gl/docs/faq/features/sharing/sharing
